OpenNefia.Content/Resources/keybinds.yml

```diff
@@ -217,6 +229,12 @@ binds:
   canFocus: true
   canRepeat: true
   repeatMode: None
+- function: UIBacklog
```

Comment:
Should also bind this to forward slash on regular keyboards. Some players like me don't have numpads.

Reply:
changed in 9eb449f